config/initializers/assets.rb

```diff
@@ -0,0 +1,2 @@
+# Precompile datetimepicker and moment dependency, to load after hard coded bootstrap script in application layout.
+Rails.application.config.assets.precompile += %w( moment.js bootstrap-datetimepicker.js )
```

Comment:
@flah00 Actually no, I got this out of the official guide: http://guides.rubyonrails.org/asset_pipeline.html (CTRL + F for "Rails.application.config.assets.precompile"):

> If you have other manifests or individual stylesheets and JavaScript files to include, you can add them to the precompile array in config/initializers/assets.rb:

Perhaps its a recently updated convention?